github/ci/prometheus/templates/token-counter.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: token-counter
+  labels:
+    app: token-counter
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: token-counter
+    spec:
+      nodeSelector:
+        type: vm
+      containers:
+      - name: token-counter
+        args:
+        - --stderrthreshold=0
+        - --prom-only
+        - --token=/github-tokens/oauth
+        image: kubevirtci/token-counter:v20190222-092216
```

Comment:
We can use the prow token-counter again once https://github.com/kubernetes/test-infra/pull/11447 is merged.